Documentation/operations/upgrade.rst

```diff
@@ -330,6 +332,9 @@ Helm Options
   please rely on Cilium's metrics to monitor status of endpoints. Example metrics include: ``cilium_policy``, ``cilium_policy_endpoint_enforcement_status``,
   ``cilium_controllers_failing`` and ``cilium_endpoint_state``.
   More detailed information about specific endpoint status information is still available through ``cilium-dbg endpoint get``.
+* The deprecated helm option ``remoteNodeIdentity`` has been removed. This should have no impact which used the previous default
+  value of ``true``: Remote nodes will now always use `remote-node` identity. If you have network policies based on
```

Comment:
Docs build is failing due to `remote-node` not using double quotes:

    `remote-nodes` -> ``remote-nodes``